Modules/ThirdParty/VNLInstantiation/src/libraryplaceholder.cxx

```diff
@@ -0,0 +1,8 @@
+
+/* This file has one symbol that is exported in order to create a valid
+ * library for backwards compatibility in linking applications that may
+ * depend on the existence of the VNLInstantiation library
+ */
+
+extern bool dummy_placeholder_variable_to_avoid_empty_library_warnings;
```

Comment:
Wouldn't this create undefined symbol for `dummy_...` if/when somebody does link to `ITKVNLInstantiation`?

Reply:
If we only had an extern declaration then you would be correct, but line 8 defines the symbol.